SonarScanner.Shim/PropertiesFileGenerator.cs

```diff
@@ -268,8 +268,19 @@ private static AnalysisProperties GetAnalysisPropertiesToWrite(AnalysisConfig co
             AnalysisProperties properties = new AnalysisProperties();
 
             properties.AddRange(config.GetAnalysisSettings(false).GetAllProperties()
-                // Strip out any sensitive properties
-                .Where(p => !p.ContainsSensitiveData()));
+                      // Strip out any sensitive properties
```

Comment:
There's a minor formatting change here, which is not relevant for this ticket. Do you use any formatting extensions? I don't mind, but then all of us should.